package.json

```diff
@@ -35,6 +37,7 @@
     "gulp-less": "~1.2.3",
     "gulp-replace": "~0.3.0",
     "protractor": "~0.24.0",
-    "http-server": "~0.6.1"
+    "http-server": "~0.6.1",
+    "jpm": "^1.0.0"
```

Comment:
@DrewML I tried to use only stable APIs from the addon-sdk, so from a code "point of view" this should work on older version of Firefox too.

Said that, when jpm builds the xpi archive, it puts a "bootstrap.js" loader inside the xpi which is not compatible with Firefox versione older than 38.

I'm going to ensure this package build correctly even with ```cfx``` (the previous xpi build tool), which generates xpis compatible with older versions.